Dockerfile.rhel7

```diff
@@ -2,12 +2,15 @@ FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20 AS
 WORKDIR /go/src/github.com/openshift/cluster-kube-controller-manager-operator
 COPY . .
 RUN make build --warn-undefined-variables
+RUN make tests-ext-build --warn-undefined-variables \
+    && gzip cluster-kube-controller-manager-operator-tests-ext
 
 FROM registry.ci.openshift.org/ocp/4.20:base-rhel9
 RUN mkdir -p /usr/share/bootkube/manifests/bootstrap-manifests/ /usr/share/bootkube/manifests/config/ /usr/share/bootkube/manifests/manifests/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-controller-manager-operator/bindata/bootkube/bootstrap-manifests /usr/share/bootkube/manifests/bootstrap-manifests/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-controller-manager-operator/bindata/bootkube/config /usr/share/bootkube/manifests/config/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-controller-manager-operator/bindata/bootkube/manifests /usr/share/bootkube/manifests/manifests/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-controller-manager-operator/cluster-kube-controller-manager-operator /usr/bin/
+COPY --from=builder /go/src/github.com/openshift/cluster-kube-controller-manager-operator/cluster-kube-controller-manager-operator-tests-ext.gz /usr/bin/
```

Comment:
Dockerfile.rhel7 is used for production. Does the new binary needs to be part of a production image?

Reply:
@ingvagabund Yes, it is for this https://docs.google.com/document/d/1cFZj9QdzW8hbHc3H0Nce-2xrJMtpDJrwAse9H7hLiWk/edit?tab=t.0#heading=h.v059hn4ylc6b
 CC @wangke19